src/res/translation/translation_sv_SE.ts

```diff
@@ -1165,47 +1165,48 @@
+        <translatorcomment>ASIO enhetsinställningar</translatorcomment>
+        <translation>Om buffertfördröjningsinställningarna är inaktiverade är det förbjudet för ljuddrivrutinen att ändra denna inställning inifrån programvaran. På Windows trycker du på knappen ASIO enhetsinställningar för att öppna panelen för drivrutinsinställningar. På Linux använder du Jack-konfigurationsverktyget för att ändra buffertstorleken.</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="177"/>
         <source>Sound card driver settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Ljudkortsinställningar</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="178"/>
         <source>This opens the driver settings of your sound card. Some drivers allow you to change buffer settings, others like ASIO4ALL let you choose input or outputs of your device(s). More information can be found on jamulus.io.</source>
-        <translation type="unfinished"></translation>
+        <translation>Detta öppnar drivrutinsinställningarna för ditt ljudkort. Vissa drivrutiner låter dig ändra buffertinställningar, andra som ASIO4ALL låter dig välja in- eller utgångar från din enhet (er). Mer information finns på jamulus.io.</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="183"/>
         <source>Opens the driver settings. Note: </source>
-        <translation type="unfinished"></translation>
+        <translation>Öppnardrivrutinsinställningarna. Notera: </translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="184"/>
         <source> currently only supports devices supporting a sample rate of </source>
-        <translation type="unfinished"></translation>
+        <translation> stöder för närvarande endast enheter som stöder en samplingsfrekvens på </translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="185"/>
         <source>Hz. You will not be able to select a driver/device which doesn&apos;t. For more help see jamulus.io.</source>
-        <translation type="unfinished"></translation>
+        <translation>Hz. Du kommer inte att kunna välja en drivrutin/enhet som inte gör det. För mer hjälp se jamulus.io.</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="199"/>
         <source>ASIO Device Settings push button</source>
-        <translation type="unfinished"></translation>
+        <translation>ASIO enhetsinställningsknapp</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="294"/>
         <source>ASIO Device Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>ASIO enhetsinställningar</translation>
```

Comment:
Not sure since I don't speak swedish. But is this a translation of ASIO **Device** Setup? We decided to use "ASIO Device Setup" to make it clearer mainly for newbies (using ASIO4ALL) that this opens the settings dialog for their audio devices/sound card etc. if somebody doesn't know about ASIO adding device here might be clearer.

Reply:
I think we should decide if we like to translate it all or partly. I personally think a mix is a bad idea. Everything else is translated and my suggestion to continue to have it like that.

Reply:
Ok. Let's wait for some others to comment here. Personally I'd prefer to have ASIO Device Setup for everything what was ASIO Setup before.

Reply:
> Ok. Let's wait for some others to comment here. Personally I'd prefer to have ASIO Device Setup for everything what was ASIO Setup before.

Do you think it should be verbatim (i.e. untranslated)?
I agree it makes sense to distinguish between "device settings" and just "settings", even in translations. Google Translate lists "enhetsinställningar" for "device settings", so in my opinion this is fine?

Reply:
Ah ok. Sorry. Probably I just misunderstood it here.